test/host/hls/test.cpp

```diff
@@ -143,6 +143,47 @@ std::unique_ptr<ACCL::ACCL> test_vadd_put(options_t options) {
     return accl;
 }
 
+void test_copy(ACCL::ACCL& accl, options_t options) {
+    //run test here:
+    //initialize a CCLO BFM and streams as needed
+    hlslib::Stream<command_word> callreq, callack;
+    hlslib::Stream<stream_word, 512> data_cclo2krnl("cclo2krnl"), data_krnl2cclo("krnl2cclo");
+
+    std::vector<unsigned int> dest = {0};
+
+    CCLO_BFM cclo(options.start_port, rank, size, dest, callreq, callack, data_cclo2krnl, data_krnl2cclo);
```

Comment:
I think this is not applicable on hardware. When I run it with the axis3x design for U55c it tries to connect to a non-existing ZMQ server and the test crashes with `CCLO @0x0: during copy the following error(s) occured: DMA DECODE ERROR, DMA NOT OKAY ERROR (00000000000000000000010100)`.

Reply:
I fixed the BFM creation for hardware builds. But I get the same error during hardware execution. Maybe we need a recent build to test this. It works in the simulator though...